### ISSUE TITLE
Refactor `example-app` to standalone

### ISSUE THREAD

**Issue by markostanimirovic** [ngrx/platform#4417]
### Information

The `example-app` project from the NgRx monorepo currently uses modules and should be refactored to standalone APIs.

### Documentation page

https://github.com/ngrx/platform/tree/main/projects/example-app

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by rainerhahnekamp** [comment]
Yes, I'd like to do that one. Can you also give me a deadline ;)

**Comment by markostanimirovic** [comment]
Thanks Rainer! There is no strict deadline, but it would be great if we could do it by the end of this month.

**Comment by sheikalthaf** [comment]
@rainerhahnekamp PR for this #4487
